proposals/rkt.adoc

```diff
@@ -0,0 +1,81 @@
+
+The core execution unit of rkt is the _pod_, a collection of one or more applications executing in a shared context. (rkt's pods are synoymous with the concept in the Kubernetes orchestration system). rkt allows users to apply different configurations (like isolation parameters) at both pod-level and at the more granular per-application level. rkt's architecture means that each pod executes directly in the classic Unix process model (i.e. there is no central daemon), in a self-contained, isolated environment.
+
+rkt's daemonless model makes it highly suitable for integration with process managers like systemd, or for embedding in agents like the kubelet or Mesos agent. Its "staged" architecture - separating the user interface from its containerisation phase - allows it to support different isolation technologies: the mainstream rkt project includes implementations like QEMU and LKVM, as well as the default "classic" Linux containers implementation of cgroups and namespaces. Furthermore, while rkt development is currently targeted primarily at GNU/Linux distributions, the staged architecture makes it possible to extend support to other platforms without fundamentally changing rkt's external interfaces.
+
+rkt strives for broad compatibility with existing container images: as well as being able to execute images conforming to the appc specification (ACIs), it also supports running Docker images natively, including Docker registry support. rkt developers closely follow the evolution of the Open Container Initiative (OCI) project, with the intent of [full native support](https://github.com/coreos/rkt/blob/v1.25.0/Documentation/proposals/oci.md#oci-image-format-roadmap) of the OCI runtime and image specifications as they reach their respective stable (1.0) milestones. rkt already includes preliminary support for working with the runtime candidate releases of both specifications. In keeping with the opinionated and scoped nature of the project, rkt does not include any native workflows for building container images, but is rather expected to be used in conjunction with build systems such as the [Dockerfile](https://docs.docker.com/engine/reference/builder/), [acbuild](https://github.com/containers/build), or [box](https://github.com/erikh/box) projects.
+
+Since its introduction by CoreOS in December 2014, the rkt project has greatly matured and is widely used.
+
+
+*Sponsor / Advisor from TOC*: Brian Grant
+
+*Proposed Maturity Level*: Incubating
+
+*Unique Identifier*: rkt
+
+*License*: Apache License v2
+
+*Source control repositories*:
+
+* https://github.com/coreos/rkt (hoping to move to github.com/rkt/rkt, pending availability of that GH organisation)
+
+*Initial Committers*:
+
+Core: @alban, @philips, @squeed, @dgonyeo, @euank, @iaguis, @jonboulle, @krnowak, @lucab, @s-urbaniak, @steveeJ, @yifan-gu
+
+*Infrastructure requirements*: CI might be useful; currently CI is sponsored by CoreOS <https://jenkins-rkt-public.prod.coreos.systems/#>
```

Comment:
Yes, because rkt is complicated :-). Since it's very invasive to a system, we want to do tests in VMs; further, we test on different OS platforms (e.g. Debian/Fedora/etc) and hence spin up EC2 instances for each run.